targets/TARGET_STM/rtc_api.c

```diff
@@ -389,6 +389,7 @@ void rtc_set_wake_up_timer(timestamp_t timestamp)
     }
 
     RtcHandle.Instance = RTC;
+    HAL_RTCEx_DeactivateWakeUpTimer(&RtcHandle);
```

Comment:
Is this really needed as rtc_deactivate_wake_up_timer function was already called in lp_ticker_set_interrupt before calling rtc_set_wake_up_timer?

Reply:
Yes, rtc_deactivate_wake_up_timer by itself is not sufficient. If HAL_RTCEx_DeactivateWakeUpTimer is removed then this test will fail.